core/src/test/java/org/testcontainers/containers/ParsedDockerComposeFileValidationTest.java

```diff
@@ -76,27 +77,43 @@ public void shouldIgnoreUnknownStructure() {
     public void shouldObtainImageNamesV1() {
         File file = new File("src/test/resources/docker-compose-imagename-parsing-v1.yml");
         ParsedDockerComposeFile parsedFile = new ParsedDockerComposeFile(file);
-        assertEquals("all defined images are found", Sets.newHashSet("redis", "mysql", "postgres"), parsedFile.getDependencyImageNames()); // redis, mysql from compose file, postgres from Dockerfile build
+        Assertions.assertThat(parsedFile.getDependencyImageNames())
```

Comment:
Formatting it so that each entry starts at the new line might improve readability.

Reply:
Formated